### PR TITLE
[#159] Run standard formatter

### LINKED ISSUE #159: Non uniform javascript formatting style
As mentioned [here](https://github.com/space-concordia-robotics/robotics-prototype/wiki/Code-Formatting-and-Conventions#javascript) in the wiki, all javascript should follow the standard conventions.

To setup this auto-formatter automatically in atom see [here](https://github.com/space-concordia-robotics/robotics-prototype/#atom)

### PR DESCRIPTION
**Pull request by PeterGhimself**
The only drawback I have found is that this standard formatter removes the parentheses we have added for boolean conditions in ternary statements. :eyes:

Closes #159

### CODE REVIEW

**Comment by vashmata** [comment]
I believe if the parentheses are removed it actually causes undesired behaviour. You should test this and check if it's an issue

**Comment by PeterGhimself** [comment]
@vashmata Unless we have some ternary statements conditions anywhere (I just checked and didn't find any candidates) it shouldn't affect anything. `x == y` will return a boolean value either way. I just like adding parentheses because it makes it a bit more readable, which I think is the same reason they used this convention at my first internship.

Via chrome console:
```
> let test = NaN == NaN
< undefined
> test
< false
> test = (NaN == NaN)
< false
```

File: robot/basestation/static/js/arm.js [hunk 1]
```diff
@@ -92,7 +92,9 @@ $(document).ready(function () {
 
   $('#toggle-arm-listener-btn').on('click', function (event) {
     event.preventDefault()
-    let serialType = $('#serial-type').text().trim()
+    let serialType = $('#serial-type')
+      .text()
+      .trim()
```
Comment (Davidster) [inline robot/basestation/static/js/arm.js]: wtf

File: robot/basestation/static/js/roslibjs/ros-helpers.js [hunk 2]
```diff
@@ -392,7 +392,14 @@ function requestSerialCommand (command, callback) {
...
 }
-function requestTask (reqTask, reqStatus, buttonID, callback, reqArgs = '', timeout = REQUEST_TIMEOUT) {
+function requestTask (
+  reqTask,
+  reqStatus,
+  buttonID,
+  callback,
+  reqArgs = '',
+  timeout = REQUEST_TIMEOUT
+) {
```
Comment (Davidster) [inline robot/basestation/static/js/roslibjs/ros-helpers.js]: wtf

File: robot/basestation/static/js/arm.js [hunk 3]
```diff
@@ -104,8 +106,11 @@ $(document).ready(function () {
     else if ($('#toggle-arm-listener-btn').is(':checked')) {
       // validate UART mode options are correct, let pass if USB mode selected
       if (
-        ($('button#mux').text().includes('Arm') && serialType == 'uart')
-          || serialType == 'usb'
+        ($('button#mux')
+          .text()
+          .includes('Arm') &&
+          serialType == 'uart') ||
+        serialType == 'usb'
```
Comment (Davidster) [inline robot/basestation/static/js/arm.js]: wtf
wtf

File: robot/basestation/static/js/arm.js [hunk 4]
```diff
@@ -182,14 +187,18 @@ $(document).ready(function () {
...
+      if (msgs[0]) {
+        $(this.id)[0].checked = !isOpen
+      } else {
+        $(this.id)[0].checked = isOpen
+      }
     }
-    sendRequest("Arm", 'motor ' + num + ' loop ' + (isOpen) ? 'open' : 'closed', armReq)
+    sendRequest(
+      'Arm',
+      'motor ' + num + ' loop ' + isOpen ? 'open' : 'closed',
```
Comment (vashmata) [inline robot/basestation/static/js/arm.js]:
```
num = 5
isOpen = true
'motor ' + num + ' loop ' + isOpen ? 'open' : 'closed'
```
expected result:
`'motor 5 loop open'`
actual result:
`'open'`

**Comment by vashmata** [comment]
Looks like that really is the only place with a ternary statement in the code. There are one or two other spots but they're commented out. I think there are spots in which ternary statements used to exist but were replaced with wordier code.

File: robot/basestation/static/js/arm.js (hunk 4 above)
Comment (PeterGhimself) [reply robot/basestation/static/js/arm.js]:
```
> 'motor ' + num + ' loop ' + (isOpen) ? 'open' : 'closed'
< "open"
```
Was this problem ever really solved though?
Enclosing parentheses around the entire ternary statement (not just the boolean condition) seems to do the trick though.
```
> 'motor ' + num + ' loop ' + (isOpen ? 'open' : 'closed')
< "motor 5 loop open"
```

Comment (PeterGhimself) [reply robot/basestation/static/js/arm.js]: Fixed

**Pull request merged**